pkg/kubelet/volumes.go

```diff
@@ -229,6 +231,15 @@ func (kl *Kubelet) getPodVolumesFromDisk() map[string]cleanerTuple {
 		volumes, err := kl.getPodVolumes(podUID)
 		if err != nil {
 			glog.Errorf("%v", err)
```

Comment:
@ncdc - I'm wondering if we can just remove it all together, it's really just duplicating the error that has already been logged in getPodVolumes